<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" alignment="Qt::AlignTop">
     <widget class="QPushButton" name="btnConnexion">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="lblMotDePasse">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="btnFermer">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtMotDePasse">
      <property name="minimumSize">
       <size>
        <width>357</width>
        <height>30</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Entrez votre mot de passe</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLineEdit" name="txtNomUtilisateur">
      <property name="minimumSize">
       <size>
        <width>357</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Entrez votre nom d'utilisateur</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignVCenter">
     <widget class="QLabel" name="lblTitre">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#lblTitre {
    background-color: #2e2e2e; /* gris fonc√© */
    color: white;               /* texte blanc */
    font-size: 24px;
    font-weight: bold;
    qproperty-alignment: 'AlignCenter'; /* centrer le texte */
    padding: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Videoclub | La Cassette</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="lblNomUtilisateur">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom d'utilisateur :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
